app/assets/javascripts/application.js.erb

```diff
@@ -484,4 +484,23 @@ $(document).ready(function() {
     $("#story_guidelines").toggle();
     return false;
   });
+
+  $(document).on("focus", "textarea#comment", function() {
+    var trigger = false;
+
+    $(".comment_form_container").on("keydown", function(e) {
```

Comment:
This is a fair concern. I apologize for not considering it. After some research, I came up with a cleaner method.